frontend/src/scenes/insights/ActionFilter/ActionFilterDropdown.tsx

```diff
@@ -83,14 +84,20 @@ export function ActionFilterDropdown({
                         key: EntityTypes.ACTIONS + action.id,
                         name: action.name,
                         volume: action.count,
+                        id: action.id,
                         action,
                     })),
                     renderInfo: function actions({ item }) {
                         return (
                             <>
                                 <AimOutlined /> Actions
+                                <Link to={`/action/${item.id}`}>
+                                    <Button type="primary" style={{ float: 'right' }}>
```

Comment:
I think this would be nicer as _just_ `edit` (also lowercase) rather than the button

Reply:
got it  will update

Reply:
updated the edit option any further changes required ?